content/en/news/2019/announcing-1.3.5/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Announcing Istio 1.3.5
+description: Istio 1.3.5 release announcement.
+publishdate: 2019-11-11
+attribution: The Istio Team
+subtitle: Minor Update
+release: 1.3.5
+aliases:
+    - /news/announcing-1.3.5
+---
+
+This release includes bug fixes to improve robustness. This release note describes what's different between Istio 1.3.4 and Istio 1.3.5.
+
+{{< relnote >}}
+
+## Bug fixes
+
+- **Fixed** Envoy listener configuration for TCP headless services. ([Issue #17748](https://github.com/istio/istio/issues/17748))
+- **Fixed** an issue which caused stale endpoints to remain even when a deployment was scaled to 0 replicas. ([Issue #14436](https://github.com/istio/istio/issues/14336))
+- **Fixed** Pilot to no longer crash when an invalid Envoy configuration is generated. ([Issue 17266](https://github.com/istio/istio/issues/17266))
+
```

Comment:
Need 2 more items for https://github.com/istio/istio/pull/18620:
```
* **Fixed** an issue with the `destination_service_name` label not getting populated for TCP metrics related to BlackHole/Passthrough clusters. ([Issue 17271](https://github.com/istio/istio/issues/17271))
* **Fixed** an issue with telemetry not reporting metrics for BlackHole/Passthrough clusters when fall through filter chains were invoked. This occurred when explicit `ServiceEntries` were configured for external services. ([Issue 17759](https://github.com/istio/istio/issues/17759))
```
CC @nrjpoddar

Reply:
Added. Thanks.

Reply:
@brian-avery I don't see the update. Did you push your updated branch?